src/VisualStudio/Core/Def/Implementation/LanguageClient/RazorInProcLanguageClient.cs

```diff
@@ -27,6 +27,7 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.Razor.Lsp
     /// </summary>
     [ContentType(ContentTypeNames.CSharpContentType)]
     [ClientName(ClientName)]
+    [RunOnContext(RunningContext.RunOnHost)]
```

Comment:
This is the fix, the rest is in support of this line compiling successfully

Reply:
can you doc what this means?  it is greek to me.

Reply:
also, why is it not on other impls of AbstractInProcLanguageClient?  And can you doc that as well?